doc/rvv-intrinsic-spec.adoc

```diff
@@ -431,6 +431,8 @@ Types with subscript "^\*^" are available when `ELEN >= 64` (that is, unavailabl
 [[floating-point-type]]
 === Floating-point types
 
+NOTE: This specification uses `_Float16` to designate IEEE-754 binary16, `float` to designate IEEE-754 binary32 and `double` to designate IEEE-754 binary64.
```

Comment:
Do we also need bfloat16?

Reply:
BF16 was not part of the original vector extensions. We will add it at a later date, once we get the base vector intrinsics ratified. Similarly for the vector crypto stuff. (Arguably, we could drop the FP16 stuff too, since Zvfh wasn't part of the original extension.)

Reply:
Got it, thanks @nick-knight !